tmt/steps/provision/__init__.py

```diff
@@ -392,11 +393,12 @@ def push(self, source=None, destination=None, options=None):
         which are '-Rrz --links --safe-links --delete'.
         """
         if options is None:
-            options = "-Rrz --links --safe-links --delete"
+            options = "-rRrz --links --safe-links --delete"
         if destination is None:
             destination = "/"
         if source is None:
-            source = self.parent.plan.workdir
+            source = f"{self.parent.plan.workdir} " \
+                     f"{self.parent.plan.run.workdir_tree}"
```

Comment:
Why there're two paths? Please clarify how it is handled?
I assume rsync simply copies both paths [folders] into the destination [folder]. Do we assume that `source` has no spaces / is shell-safe?

Also, there's no check for trailing `/` which copies only the contents of the folder (`rsync` behavior). Is that expected?

Same for the `destination`, actually.

Reply:
yeah, this is not important anymore, we sync only workdir as before, thus resolving.

In the reviewed patch set this tried to sync multiple directories without a common parent, thus 2 paths ... but yeah, as this is changed back now, I am resolving ...